lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/body.txt

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-{% trans "You’re almost there!" %}
+{% trans "You're almost there!" %}
```

Comment:
:(  What's this about?

Reply:
i was seeing a random space showing up after that unicode character for some reason on my terminal making it look like
You' re
which was odd, but maybe that's just a weird terminal thing. I can change it back if you want

Reply:
changed back

Reply:
Doesn't look like you pushed the revert back to github though. :shrug: